CHANGES.rst

```diff
@@ -6,6 +6,10 @@ New Features
 
 API Changes
 ^^^^^^^^^^^
+Previously, the ``setup`` and ``teardown`` methods were run only once
+even when the benchmark had ``repeat > 1`` specified. This is now
+changed so that they are run on every repeat, and additionally before
+and after profiling runs.
```

Comment:
The profiling runs previously redo the setup before profiling (the benchmark is run once before the profiling).